src/Extension/FluentSiteTreeExtension.php

```diff
@@ -43,15 +42,6 @@ class FluentSiteTreeExtension extends FluentVersionedExtension
      */
     private static $locale_published_status_message = true;
 
-    /**
-     * Enable localise actions (copy to draft and copy & publish actions)
-     * these actions can be used to localise page content directly via main page actions
-     *
-     * @config
-     * @var bool
-     */
-    private static $localise_actions_enabled = true;
```

Comment:
Oops, did this config get deleted? This has become a breaking change if so.

Reply:
Also, I'm not 100% sure what this config is doing. It's possible I just need to be educated and nothing at all is wrong. :)